library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -564,6 +557,9 @@ libspdm_return_t libspdm_build_response(void *context, const uint32_t *session_i
             LIBSPDM_DEBUG((LIBSPDM_DEBUG_ERROR,
                            "Warning: Could not save chunk. Scratch buffer too small.\n"));
 
+            /* Should we really be returning LARGE_RESPONSE is we aren't supporting chunking?
```

Comment:
`LargeResponse` is the correct error code when `DataTransferSize < Response <= MaxSPDMmsgSize`. `ResponseTooLarge` is the correct error code when `Response > MaxSPDMmsgSize`. I'll file an issue with the specification to clarify this.

Reply:
https://github.com/DMTF/Security-TF/issues/2060

Reply:
Although I think if a endpoint does not support chunking then it should set `DataTransferSize == MaxSPDMmsgSize` in which case it would be `ResponseTooLarge`.